modules/cats/build.sbt

```diff
@@ -1,8 +1,10 @@
 name := "pureconfig-cats"
 
+val catsVersion = "1.4.0"
+
 libraryDependencies ++= Seq(
-  "org.typelevel" %% "cats-core" % "1.1.0",
-  "org.typelevel" %% "cats-laws" % "1.1.0" % "test",
+  "org.typelevel" %% "cats-core" % catsVersion,
```

Comment:
Nitpick: can you leave the versions defined in-place as before? Since it is only used twice near one another, I think it becomes more readable if we just write the same thing twice.

Reply:
Sure, if that works better for you I'll inline :)